static/lang/re-en.json

```
@@ -5330,6 +5330,9 @@
                 "Text": "If you hit with your jaws and deal damage, you gain [[/r (ceil(@item.level/2))d6 #Temporary Hit Points]] temporary Hit Points."
             },
             "SpellEffectMantle": {
+                "DamageNote": {
+                    "HeartOfIce": "You take @Damage[2d6[cold]] damage"
```

```suggestion
                    "HeartOfIce": "The attacker takes @Damage[2d6[cold]] damage"
```
Maybe more clear wording